### ISSUE TITLE
Add theming support

### ISSUE THREAD

**Issue by osdiab** [osdiab/spinach-web-app#1]
Emotion theming: https://emotion.sh/docs/theming

**Comment by AndyMoreland** [comment]
Is intent to theme per-client?

**Comment by AndyMoreland** [comment]
s/client/intermediary

**Comment by osdiab** [comment]
yep per intermediary (i.e. Vistra) - emotion stuff would pertain mostly to color scheme, there's also some other things like logo, and copy around the site like the name of the client

we can start off simply by just having it hard-coded in variables no need to do anything heavier than that.

**Comment by osdiab** [comment]
This couled also depend on #4 ; if we do separate infrastructure per intermediary, then maybe it makes sense to just encode this kinda stuff in environment variables; or maybe it makes sense to have it be in the DB but cached so that we dont need to do DB queries every time we want to show a page.

**Comment by osdiab** [comment]
we're not doing separate infra per intermediary, lets store stuff in the DB for each vendor's config
